ivy/functional/frontends/numpy/__init__.py

```diff
@@ -601,13 +601,17 @@ def promote_types_of_numpy_inputs(
 from ivy.functional.frontends.numpy.linalg.matrix_and_vector_products import _matmul
 
 from ivy.functional.frontends.numpy.mathematical_functions.extrema_finding import (
-    _maximum,
     _minimum,
-    _fmax,
 )
 
+
+frontend_array = array
```

Comment:
I think this is old changes from old commit, please also remove these (repetition with the existing code as well)